### ISSUE TITLE
The Deep Kernel Learning Notebook from readthedeocs.io - How to incorporate a combination kernel instead of a single kernel in the existing codebase?

### ISSUE THREAD

**Issue by GautamV234** [JaxGaussianProcesses/GPJax#210]
### Discussed in https://github.com/orgs/JaxGaussianProcesses/discussions/209

<div type='discussions-op-text'>

<sup>Originally posted by **GautamV234** April  5, 2023</sup>
The following is the notebook being referred to in the below discussion - https://gpjax.readthedocs.io/en/latest/examples/haiku.html#Deep-Kernel-Learning 

Hi all, My Doubt is regarding the utilization of a kernel combination such as jk.RBF() + jk.RBF() instead of a single kernel and keep the remaining architecture the same i.e. same neural network configuration in Haiku and the same way of incorporation of the linear weights in the parameter state. When I try to replace the jk.RBF() with the combination jk.RBF() + jk.RBF(), I get the following error in calculating the gradient of the MLL
![image](https://user-images.githubusercontent.com/68188848/229962496-b198e942-bce7-427f-8087-180dae54efbe.png)

It would be great if someone could please clarify how to go about using a combination kernel along with additional parameters such as the linear weights and biases in this case. Thanks a lot! </div>

**Comment by thomaspinder** [comment]
Hi @GautamV234, thanks for raising this. We recently released v0.6 which revamps this functionality (see the notebook [here](https://docs.jaxgaussianprocesses.com/examples/deep_kernels/)). I believe this issue should now be resolved; could you please confirm if your issue has been resolved?
